npm/src/aws-config.ts

```diff
@@ -1,32 +1,32 @@
-import AWS from "aws-sdk"
+import AWS, { Credentials } from "aws-sdk"
 import { IFrontEndInfo } from "."
 
 export const configureAws: (frontEndInfo: IFrontEndInfo) => void = (
   frontEndInfo: IFrontEndInfo
 ) => {
-  if (frontEndInfo.cognitoEndpointOverride) {
-    overrideCognitoEndpoint(frontEndInfo.cognitoEndpointOverride)
-  }
-  if (frontEndInfo.s3EndpointOverride) {
-    overrideS3Endpoint(frontEndInfo.s3EndpointOverride)
-  }
-}
-
-const overrideCognitoEndpoint: (cognitoEndpointOverride: string) => void = (
-  cognitoEndpointOverride: string
-) => {
-  console.log(`Overriding Cognito endpoint with '${cognitoEndpointOverride}'`)
-  AWS.config.cognitoidentity = {
-    endpoint: cognitoEndpointOverride
+  if (frontEndInfo.uploadUrl) {
+    overrideS3Endpoint(frontEndInfo.uploadUrl)
   }
 }
 
 const overrideS3Endpoint: (s3EndpointOverride: string) => void = (
   s3EndpointOverride: string
 ) => {
-  console.log(`Overriding S3 endpoint with '${s3EndpointOverride}'`)
+  const endpoint = new AWS.Endpoint(s3EndpointOverride)
+  AWS.config.update({
+    credentials: new Credentials({
+      // The AWS SDK requires credentials to generate a request, but we are not using them for authentication so set
+      // them to placeholder values
+      accessKeyId: "some-placeholder-id",
```

Comment:
Maybe change to consistent naming by removing "some" from the value

Reply:
Makes sense, I've removed it.